packages/ui5-middleware-simpleproxy/lib/proxy.js

```diff
@@ -17,17 +25,30 @@ const log = require("@ui5/logger").getLogger("server:custommiddleware:proxy");
  * @returns {function} Middleware function to use
  */
 module.exports = function({resources, options}) {
+    // Environment wins over YAML configuration when loading settings
+    const providedBaseUri = env.baseUri || (options.configuration && options.configuration.baseUri);
+    // Handle fact that `strictSSL` is a boolean so we have to do existence checks
+    const falseyStrictSSL = 
+      (
+        env.strictSSL === undefined 
```

Comment:
`env.stricSSL` will never be `undefined` as `strictSSL: !(process.env.UI5_MIDDLEWARE_SIMPLE_PROXY_STRICT_SSL === "false")` will be `true` in case of "`UI5_MIDDLEWARE_SIMPLE_PROXY_STRICT_SSL=undefined`". That's why I made this function check in my suggestion.

Reply:
Yup, you're correct as I mentioned privately yesterday. I'm going to take the time to do this right, but probably can't until this weekend. I will withdraw this to draft status for the time being.

Reply:
Hi Ethan, is there anything I can help you with? BR, Peter

Reply:
@petermuessig Just more time in the day. Can you help me with that? 😂 Hope to get this done today or tomorrow.

Reply:
Looks good now. THX. :-)